src/bootstrap/compile.rs

```diff
@@ -1106,11 +1106,11 @@ impl Step for Sysroot {
         let compiler = self.compiler;
         let host_dir = builder.out.join(&compiler.host.triple);
         let sysroot = if compiler.stage == 0 {
-            host_dir.join("stage0-sysroot")
+            builder.out.join(&compiler.host.triple).join("stage0-sysroot")
```

Comment:
What does this achieve? It's not immediately obvious how this is relevant to #101961, though correct me if I'm wrong.

Reply:
never mind, it is one and the same thing. if you want i can revert the change

Reply:
@compiler-errors what am i missing ?

Reply:
Please re-read the description in #101961 for what changes it was asking to be made. 

Also, it's conventional to comment and/or claim the issue before putting up a fix for something, might be best to ask the current owner of that issue if they're still working on it!

Reply:
apologies. i thought the bot assigns it when you make change and it was unassigned from the current user.. BTW i have made the changes as per suggestion. Please check and let me know if needs any more changes?

Reply:
@rust-highfive claim

Reply:
@compiler-errors can you pls take a look, if this looks good ?